Sources/App/routes.swift

```diff
@@ -0,0 +1,38 @@
+import Vapor
+
+extension Application {
+    func routes() throws {
+        get { req async throws -> View in
+            if req.session.data["playground"] == "true" {
+                return try await req.view.render("playground")
+            } else {
+                return try await req.view.render("index")
+            }
+        }
+
+        get("privacy-policy") { req async throws -> View in
+            return try await req.view.render("privacy-policy")
+        }
+
+        // router.all
+        get("letsplay") { req -> Response in
+            req.session.data["playground"] = "true"
+            return req.redirect(to: "/")
+        }
```

Comment:
Serving `all` HTTP methods is not supported in Vapor, so I only implemented GET.